packages/nc-gui-v2/middleware/auth.global.ts

```diff
@@ -33,6 +33,9 @@ export default defineNuxtRouteMiddleware((to, from) => {
 
   /** if auth is required or unspecified (same as required) and user is not signed in, redirect to signin page */
   if ((to.meta.requiresAuth || typeof to.meta.requiresAuth === 'undefined') && !state.signedIn.value) {
+    if (state.appInfo?.value?.firstUser) {
```

Comment:
Missing documentation, add a comment what this section does
`/** if this is the first user, direct straight to signup */` or something like that